docs/database-functionalities/query-modules/implement-query-modules.md

```diff
@@ -145,6 +145,44 @@ return mgp.Record(args=args_copy, vertex_count=vertex_count,
                   avg_degree=avg_degree, props=props)
 ```
 
+### Writeable procedure
+
+Writeable procedures can be implemented in very similar way as read-only
```

Comment:
```suggestion
Writeable procedures can be implemented in a very similar way as read-only
```

Reply:
@jbajic had the same comment, already applied his changes.